client/ladder/ladder.tsx

```diff
@@ -141,15 +218,15 @@ export function Ladder({ matchmakingType: routeType }: LadderProps) {
         <ScrollDivider $show={true} $showAt='bottom' />
       </TabsContainer>
       <Content>
-        {rankings ? (
+        {searchResults || rankings ? (
           <LadderTable
-            lastUpdated={rankings.lastUpdated}
-            totalCount={rankings.totalCount}
-            players={rankings.players}
+            {...rankingsData}
             usersById={usersById}
-            isLoading={rankings.isLoading}
-            lastError={rankings.lastError}
-            curTime={Number(rankings.fetchTime)}
+            lastError={lastError}
+            curTime={CURRENT_TIME}
```

Comment:
You can't do this, you've built this such that this is the value when this file is loaded, not when the rankings are fetched. It won't ever change. You should probably just add this fetchTime value to both sides of the reducer, like it was previously